library/tasks/Picard.wdl

```diff
@@ -1,3 +1,43 @@
+task SortBam {
+    File bam_input
+    String sort_order = "coordinate"
```

Comment:
While this is the default order you set, `queryname` is the order we use in Optimus, am I correct?

Reply:
yes, but the majority of the use cases for this task will be by coordinate